yarn.lock

```diff
@@ -1885,7 +1885,7 @@
     "@jest/types" "^25.5.0"
     jest-mock "^25.5.0"
 
-"@jest/fake-timers@^25.1.0", "@jest/fake-timers@^25.5.0":
+"@jest/fake-timers@^25.5.0":
   version "25.5.0"
   resolved "https://registry.yarnpkg.com/@jest/fake-timers/-/fake-timers-25.5.0.tgz#46352e00533c024c90c2bc2ad9f2959f7f114185"
   integrity sha512-9y2+uGnESw/oyOI3eww9yaxdZyHq7XvprfP/eeoCsjqKYts2yRlsHS/SgjPDV8FyMfn2nbMy8YzUk6nyvdLOpQ==
```

Comment:
Are these yarn.lock changes related? If not I think you should revert prior to merging

Reply:
Will do. They are not related, but I figured someone had missed them on a previous commit. I thought the build might break, so I included them, but will remove on next rebase.